src/SFA.DAS.AssessorService.Api.Common/Settings/IAzureApiClientConfiguration.cs

```diff
@@ -1,11 +1,17 @@
-﻿namespace SFA.DAS.AssessorService.Settings
+﻿namespace SFA.DAS.AssessorService.Api.Common
 {
-    public interface IAzureApiAuthentication
+    public interface IAzureApiClientConfiguration
     {
-        string ApiBaseAddress { get; set; }
         string Id { get; set; }
+
         string Key { get; set; }
+
+        string ApiBaseAddress { get; set; }
+
         string ProductId { get; set; }
+
         string GroupId { get; set; }
+
+        string RequestBaseAddress { get; set; }
```

Comment:
@ChrisJWoodcock what is the difference between the` ApiBaseAddress` and `RequestBaseAddress` ?

Reply:
Just legacy with different develops calling things differently I tried to make as few changes as possible in this PR... there is still lots to do